### DISCUSSION TITLE
MC-224729

### DISCUSSION THREAD

**Discussion by ghost** [ACGaming/UniversalTweaks#317] [Bugfix Suggestions]
[MC-224729](https://bugs.mojang.com/browse/MC-224729)

MOD:[Chunk Saving Fix](https://www.curseforge.com/minecraft/mc-mods/chunk-saving-fix)
Chunk saving fix:

Fixes a critical bug, [https://bugs.mojang.com/browse/MC-224729](https://www.curseforge.com/linkout?remoteUrl=https%253a%252f%252fbugs.mojang.com%252fbrowse%252fMC-224729), causes worldgen happening far away from the player to be cut off and cause trees and other structures to generate halfway across chunk borders, causing big issues.

**Comment by ACGaming** [comment]
Not a 1.12.2 bug from the looks of it.
